### DISCUSSION TITLE
Keepalive on SFTP is killing my AWS budget.

### DISCUSSION THREAD

**Discussion by robert-archibald-wolfe** [Natizyskunk/vscode-sftp#231] [Q&A] [answered]
It looks like this is keeping my sftp connection with AWS S3 open, well, forever. I have run up 200+ connection hours in less than a week which to me seems a little odd :). My only explanation is that the sftp connection is staying alive the whole time that vscode is open and, well, I get it that it means that uploads are quicker, it just gets awfully expensive amazingly fast. Oh, and yes, I am using multiple machines so the multiple hours do make sense if all of them are connected all the time.
I am guessing that there is a config option that I can use to logout at least periodically, but I cannot find it.
HELP! I am going to have to stop using the plugin if I can't fix this.

**Comment by robert-archibald-wolfe** [comment, marked as answer]
and ... i am a moron ... 
forgot to shut down the servers when i was done with them . 

and I can't count ... was up for 10 days so ... 240 hours.

**Reply by Natizyskunk** [reply]
@Port-Wallis-Technologies, So is this fixed now ? Can i close this discussion or do you have another question ? 🙂
